src/assets/simp/build/simp.spec

```diff
@@ -36,9 +36,9 @@ Requires: pupmod-herculesteam-augeasproviders_puppet >= 2.2.1, pupmod-herculeste
 Requires: pupmod-herculesteam-augeasproviders_shellvar >= 3.1.0, pupmod-herculesteam-augeasproviders_shellvar < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_ssh >= 3.2.1, pupmod-herculesteam-augeasproviders_ssh < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_sysctl >= 2.3.1, pupmod-herculesteam-augeasproviders_sysctl < 3.0.0
-Requires: pupmod-onyxpoint-gpasswd >= 1.0.6, pupmod-onyxpoint-gpasswd < 2.0.0
+Requires: pupmod-onyxpoint-gpasswd >= 1.0.7, pupmod-onyxpoint-gpasswd < 2.0.0
 Requires: pupmod-puppet-yum >= 3.1.1, pupmod-puppet-yum < 4.0.0
-Requires: pupmod-puppetlabs-apache >= 3.0.0, pupmod-puppetlabs-apache < 5.0.0
+Requires: pupmod-puppetlabs-apache >= 4.0.0, pupmod-puppetlabs-apache < 5.0.0
```

Comment:
Please revert to 3.0.0.  We don't want to force our users to upgrade a major version for a module we don't use.